cmd/traefik/traefik.go

```diff
@@ -427,7 +427,7 @@ func getDefaultsEntrypoints(staticConfiguration *static.Configuration) []string
 		}
 	}
 
-	sort.Strings(defaultEntryPoints)
+	slices.Sort(defaultEntryPoints)
```

Comment:
As of Go 1.22, `sort.Strings` simply calls `slices.Sort`. See https://pkg.go.dev/sort#Strings